app/routes/content.js

```diff
@@ -6,6 +6,15 @@ import config from 'ember-get-config';
 import loadAll from 'ember-osf/utils/load-relationship';
 import permissions from 'ember-osf/const/permissions';
 
+// Error handling for API
+const handlers = new Map([
+    // format: ['Message detail', 'page']
+    ['Authentication credentials were not provided.', 'forbidden'], // 401
+    ['You do not have permission to perform this action.', 'forbidden'], // 403
```

Comment:
Really, a 401 should have a separate unauthorized page and tell the user that they need to log in in order to get either the correct page, or the forbidden page. With 403s going to not found, we're just obfuscating the fact that the project was made private. The API will still return a 403 code. It should be fixed in the API to be correct. I can change it quickly, but just want to make sure (since it's a hotfix) that we get it right.